src/worker/fastly.rs

```diff
@@ -0,0 +1,76 @@
+use anyhow::{anyhow, Context};
+use reqwest::blocking::Client;
+
+pub struct Fastly {
+    api_token: String,
+    static_domain_name: String,
+}
+
+impl Fastly {
+    pub fn from_environment() -> Option<Self> {
+        let api_token = dotenvy::var("FASTLY_API_TOKEN").expect("missing FASTLY_API_TOKEN");
```

Comment:
since this is currently not set on production yet, it might be better to assume that this env var is optional and only run the invalidation if it is set. I guess we do the same for CloudFront since the type in the environment struct is `Option<CloudFront>`.